streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql

```diff
@@ -197,6 +197,7 @@ create table if not exists `t_flink_project` (
 create table if not exists `t_flink_savepoint` (
   `id` bigint generated by default as identity not null,
   `app_id` bigint not null,
+  `chk_id` bigint default null,
```

Comment:
Plz check pgsql and mysql dialect whether lack field or without default value.

Reply:
> Plz check pgsql and mysql dialect whether lack field or without default value.

mysql and pgsql have chk_id，this is data-h2.sql bug.